stock_ux/__manifest__.py

```diff
@@ -42,6 +42,8 @@
         'views/stock_move_views.xml',
         'views/stock_backorder_confirmation_views.xml',
         'views/stock_return_picking_views.xml',
+        'view/stock_picking_view.xml',
```

Comment:
@ValentinoDefelice  here the path should be `views/stock_picking_view.xml` ,
also the name of the files should be using postfix `views` after model name
this apply for the line 46 too.